src/coreclr/interop/interoplib.cpp

```diff
@@ -123,15 +123,15 @@ namespace InteropLib
                 // interface QI. Once we have the IReferenceTracker
                 // instance we can be sure the QI for IUnknown will really
                 // be the true identity.
-                HRESULT hr = external->QueryInterface(&trackerObject);
+                HRESULT hr = external->QueryInterface(IID_IReferenceTracker,(void **) &trackerObject);
```

Comment:
Nit: Spacing is off here.

```suggestion
                HRESULT hr = external->QueryInterface(IID_IReferenceTracker, (void**)&trackerObject);
```